src/clarity-angular/data/stack-view/_stack-view.clarity.scss

```diff
@@ -44,6 +44,7 @@ $clr-stack-font-weight: clr-getTypePropertyValueForDomElement(stackview_text, fo
         border-radius: $clr-default-borderradius;
         overflow-y: auto;
         background-color: $gray-lighter;
+        word-wrap: break-word;
```

Comment:
This is `word-wrap` not `word-break`:
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

The difference is that `break-all` will break between any two letters, while with `break-word` it will try to break the words and only if the browser cannot overflow but it should, then it will break the unbreakable words.